CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+### Changed
+- Made username and password fields required on login and registration pages. #3795
```

Comment:
This isn't really meaningful to the user. This is an implementation detail of how we fixed the real issue. The issue we fixed is that error messages were inconsistent. This should be reworded and moved to fixed. It should also reference the issue, not the PR.